formats/conversion.go

```diff
@@ -141,14 +142,25 @@ func ConvertToOperationalRiskViolationScanTableRow(rows []OperationalRiskViolati
 	return
 }
 
-func ConvertToSecretsTableRow(rows []SourceCodeRow) (tableRows []secretsTableRow) {
+func ConvertToSecretsTableRow(rows []SourceCodeRow, tokenValidation bool) (tableRows []secretsTableRow) {
 	for i := range rows {
-		tableRows = append(tableRows, secretsTableRow{
+		newRow := secretsTableRow{
 			severity:   rows[i].Severity,
 			file:       rows[i].File,
 			lineColumn: strconv.Itoa(rows[i].StartLine) + ":" + strconv.Itoa(rows[i].StartColumn),
 			secret:     rows[i].Snippet,
-		})
+		}
+		if tokenValidation {
+			newRow.tokenValidation = "Not a token"
+			newRow.tokenInfo = "No Extra Info"
```

Comment:
```suggestion
			newRow.tokenValidation = "Not a token"
```
Leave empty or use `N/A`, use enums string values or const

Reply:
This texts were required by Eyal and Asaf Karas. I will move it to an enum.